code/datums/components/stationloving.dm

```diff
@@ -18,14 +18,13 @@
 		relocate()
 
 /datum/component/stationloving/RegisterWithParent()
-	RegisterSignal(parent, COMSIG_MOVABLE_Z_CHANGED, .proc/on_parent_z_change)
 	RegisterSignal(parent, COMSIG_PARENT_PREQDELETED, .proc/on_parent_pre_qdeleted)
 	RegisterSignal(parent, COMSIG_ITEM_IMBUE_SOUL, .proc/check_soul_imbue)
 	RegisterSignal(parent, COMSIG_ITEM_MARK_RETRIEVAL, .proc/check_mark_retrieval)
 	// Relocate when we become unreachable
-	RegisterSignal(parent, COMSIG_MOVABLE_SECLUDED_LOCATION, .proc/on_parent_unreachable)
+	RegisterSignal(parent, COMSIG_MOVABLE_MOVED, .proc/on_parent_moved)
 	// Relocate when our loc, or any of our loc's locs, becomes unreachable
-	var/static/list/loc_connections = list(COMSIG_MOVABLE_SECLUDED_LOCATION = .proc/on_parent_unreachable)
+	var/static/list/loc_connections = list(COMSIG_MOVABLE_MOVED = .proc/on_parent_moved)
 	AddComponent(/datum/component/connect_containers, parent, loc_connections)
```

Comment:
Might be a good idea to register for the loc gaining the trait (signal gets sent off trait add) just to future proof things